components/esp_modem/test/host_test/env.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/bash
 
-lwip=$1
-lwip_uri=$2
-lwip_contrib=$3
-
-wget --no-verbose ${lwip_uri}/${lwip}.zip
-unzip -oq ${lwip}.zip
-wget --no-verbose ${lwip_uri}/${lwip_contrib}.zip
-unzip -oq ${lwip_contrib}.zip
+idf_version=$1
+
+if [ "$idf_version" == "release-v4.3" ]; then
+  lwip=lwip-2.1.2
+  lwip_uri=http://download.savannah.nongnu.org/releases/lwip
+  lwip_contrib=contrib-2.1.0
+
+  wget --no-verbose ${lwip_uri}/${lwip}.zip
+  unzip -oq ${lwip}.zip
+  wget --no-verbose ${lwip_uri}/${lwip_contrib}.zip
+  unzip -oq ${lwip_contrib}.zip
```

Comment:
This will probably need to be added only if we need `lwip`, so for modem tests.
Suggest that we take two arguments, not only the IDF version, but also a component and decide based on it

Reply:
Ok, so you suggest to create common `env.sh` for all the tests, right?